lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -409,8 +409,24 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
     }
 
     var legendUri, legendMimeType;
-    // There can be multiple styles - just get the first if so.
+    // If style is defined in parameters, use that, but only if a style with that name can be found.
```

Comment:
I didn't realize that was an option. It does sound like a much better idea. Thanks!